### PR TITLE
TodoList App

### PR DESCRIPTION
**Pull request by mbabaliiryn**
## TodoList App
In this project;
A user is able to create and delete a project.
A user is able to create, delete and edit a todo.

### CODE REVIEW

**Comment by keshav-c** [comment]
1. There were no merge conflicts in the repo, as of the last code review request. See the image
gitlog

2. I have made the UI design much better, and it should feel better to use now.

3. The Readme (in the feature branch) has been updated with detailed instructions and information about the project.

File: src/js/form_builder/forms/project_form.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+import { openProjectTab } from '../../display/change_dom';
+
+// const validateForm = () => {
+//   const projectFormInputs = document.forms['new-project'].title.value;
+//   if (projectFormInputs === '') {
+//     alert('Title can not be empty');
+//     return false;
+//   }
+//   return projectFormInputs;
+// };
```
Comment (wathigo) [inline src/js/form_builder/forms/project_form.js:14]: * It’s better to delete **commented piece of codes**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

Comment (keshav-c) [reply src/js/form_builder/forms/project_form.js]: Thanks for pointing this out. I have removed this commented code.

**Comment by keshav-c** [comment]
I have fixed the issues with the Local Storage and removed the commented code.

**Comment by keshav-c** [comment]
Thanks!

**Pull request merged**